gui-apps/grimblast/grimblast-2022.10.13.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
Please change this to the ISC license and add your preferred name and email. An example is at https://github.com/bsd-ac/wayland-desktop/blob/master/gui-apps/fuzzel/fuzzel-9999.ebuild#L1

Reply:
My mistake, I see that's it's written plainly in the README.